frontend/src/components/layouts/layout.js

```diff
@@ -16,11 +16,11 @@ const useStyles = makeStyles((theme) => ({
     textAlign: "center",
     margin: `${theme.spacing(4)}px 0`,
   },
-  alert: {
+  alert: props => ({
     width: "100%",
     zIndex: 100,
-    marginTop: theme.spacing(-2),
-  },
+    marginTop: !props.donationCampaignRunning && theme.spacing(-2),
```

Comment:
Is this related to the error checking above ? If not maybe split into another PR...

Reply:
I think I would leave it in here because this is another bug that happens when logging out while in the chat window: You'll get redirected to /signin and the page will look weird because the donation banner goes over the message telling you that you need to log in to chat